src/lib/config-utils.js

```diff
@@ -21,27 +21,26 @@ const BLACKLIST = {
 };
 
 /**
- * Normalise user list.
+ * Normalize user list.
  * @return {Array}
  */
-export function normalizeUserlist(oldFormat: any, newFormat: any) {
+export function normalizeUserList(...args: any): Array<string> {
   const result = [];
-  /* eslint prefer-rest-params: "off" */
 
-  for (let i = 0; i < arguments.length; i++) {
-    if (arguments[i] == null) {
+  for (const value of args) {
+    if (value == null) {
       continue;
     }
-
     // if it's a string, split it to array
-    if (_.isString(arguments[i])) {
-      result.push(arguments[i].split(/\s+/));
-    } else if (Array.isArray(arguments[i])) {
-      result.push(arguments[i]);
+    if (typeof value === 'string') {
```

Comment:
I'd create an utility for use in several places `typeof value === 'string'`

Reply:
where ?